src/components/DlChart/DlChartLabels.vue

```diff
@@ -19,16 +19,48 @@
                     :data-index="index"
                     :color="labelColor"
                 >
-                    {{ label }}
+                    {{ label.title || label }}
                 </dl-typography>
                 <dl-typography
                     v-else
-                    :ref="(el) => forwardChildEl(el, index)"
+                    :ref="(el) => forwardChildEl(el, 'textRef', index)"
                     :data-index="index"
                     :size="fontSize"
                     :color="labelColor"
                 >
-                    {{ label }}
+                    {{ label.title || label }}
```

Comment:
Id prefer if we have this specifically as a computed getter instead of having it here